src/hardware/introduction.md

```diff
@@ -1,15 +1,28 @@
+Geotab supports Hardware Add-On integration through the CAN, USB, RS232, and BLE interfaces. The Add-On Protocols section outlines the integration process for all four interfaces. Geotab's open platform allows for expandability to support the unique data sets of any Hardware Add-On. GO device can currently report these [Hardware Add-On data types](../hardware-add-on-data-types/).
 
-## Input-Output Expander (IOX)
+When using Geotab's Hardware Add-On protocols, it is important not to send excessive amounts of data to the GO device.  This can be assured by only sending data when it is completely necessary and avoiding the sending of repetitive values. The GO device will stop recording data from the Hardware Add-On device if excessive Add-On data is detected. When this condition occurs, "Status data no longer being saved due to excessive data" will appear in the Fault Data for the GO device within MyGeotab. In order to re-enable data logging for the Hardware Add-On, the GO device will need to be completely disconnected from power and then powered back up again.
 
-Third-party devices communicate with the GO device using an IOX Add-On corresponding to the aforementioned interfaces, namely the [IOX-CAN](https://docs.google.com/document/d/19Wlwsb_AnpE3AndOf1cbiFLzTOCI5fwscOwQh5a6jF8/preview), [IOX-USB](https://docs.google.com/document/d/1_vocie4MYAHAUBOJ_AUXzYN11jrR5jyCynFgbqigFys/preview), [IOX-RS232](https://docs.google.com/document/d/1UDEwQOY2zH1ABQ3UP6rdlcZ--LPt5SUbKEZ_AZgUsWk/preview) and [IOX-BT](https://docs.google.com/document/d/1ICzpfhyYQEl1acQtJ9AGw09pzH5ADHNC8EGIcvTP2KU/preview).
+## Integrating with the Input-Output Expander (IOX)
 
-Third-party data sent to the GO device will be transmitted to the MyGeotab platform. Data received from the third-party device can be queried within MyGeotab by navigating to **Engine &amp; Maintenance** > **Engine &amp; Device…** > **Measurements**. It can also be reported on just like any other status data.
\ No newline at end of file
+Hardware Add-Ons communicate with the GO device using an IOX corresponding to the aforementioned interfaces, namely the [IOX-CAN](https://docs.google.com/document/d/19Wlwsb_AnpE3AndOf1cbiFLzTOCI5fwscOwQh5a6jF8/preview), [IOX-USB](https://docs.google.com/document/d/1_vocie4MYAHAUBOJ_AUXzYN11jrR5jyCynFgbqigFys/preview), [IOX-RS232](https://docs.google.com/document/d/1UDEwQOY2zH1ABQ3UP6rdlcZ--LPt5SUbKEZ_AZgUsWk/preview), [IOX-BT](https://docs.google.com/document/d/1ICzpfhyYQEl1acQtJ9AGw09pzH5ADHNC8EGIcvTP2KU/preview) and any [Custom IOX](../developing-an-iox/hardware-design-guide/). The format of data sent through an IOX could vary with the type of IOX, which is shown in the table below. All IOXs support data transfer from Add-On to GO device, while some IOXs also support data transfer from GO device to Add-On.
+
+| IOX Type | Add-On Protocol | Status Data | Custom Data | Binary Data | Data Transfer From GO Device To Add-On   |
```

Comment:
What is the Add-On Protocol?

Reply:
Hi Stephen, this refers to the "Add-On Protocols" on the side bar of the SDK page. Do you think more clarification is needed?

Reply:
I think the column should be removed after IOX-Analog is removed as all entries would have a Y.

Reply:
Hi Stephen. All your earlier review has now been addressed except deleting the "Add-On Protocols" column. I think this tells people these IOXs have protocols of their own, as opposed to other IOXs that don't. Shall we keep the column?

Reply:
If you say so.